docs/csharp/cs-dev-kit-faq.md

```diff
@@ -215,6 +215,12 @@ If you are encountering issues with Test Explorer, you can enable diagnostic log
 
 This information will help in diagnosing and resolving the issue more efficiently.
 
+### Known Issues
+
+- Using xUnit v3 with package Microsoft.Testing.Platform on macOS can cause test discovery and debugging issues in the Test Explorer for C# Devkit and Visual Studio.
```

Comment:
Is this specific to mac?

Reply:
I verified that this is also failing on VSCode C#DevKit on Windows.

As a side note. I verified that on VS it is also failing to attach the debugger.

Reply:
I mean if it is failing on all OS's I'd rather that we fix this than document it here.

Reply:
> I verified that this is also failing on VSCode C#DevKit on Windows.

It works for me on Windows. I think we need to have better understanding of what the issue is, when it's happening, so we are able to document it correctly, instead of guiding users in a wrong way.